testscommon/outportStub.go

```diff
@@ -3,37 +3,33 @@ package testscommon
 import (
 	"github.com/ElrondNetwork/elrond-go/core/statistics"
 	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/indexer"
 	"github.com/ElrondNetwork/elrond-go/data/state"
 	"github.com/ElrondNetwork/elrond-go/outport"
-	"github.com/ElrondNetwork/elrond-go/outport/types"
 )
 
 // OutportStub is a mock implementation fot the OutportHandler interface
 type OutportStub struct {
-	SaveBlockCalled             func(args types.ArgsSaveBlocks)
-	SaveValidatorsRatingCalled  func(e string, k []types.ValidatorRatingInfo)
+	SaveBlockCalled             func(args *indexer.ArgsSaveBlockData)
+	SaveValidatorsRatingCalled  func(e string, k []*indexer.ValidatorRatingInfo)
```

Comment:
e and k as parameter name? Also valid for the other functions

Reply:
refactored.